app/helpers/application_helper.rb

```diff
@@ -25,7 +25,7 @@ def relevant_prior_authority_list_anchor(prior_authority_application)
     when 'draft'
       :draft
     else
-      :assessed
+      :reviewed
```

Comment:
I had extended the '#relevant_prior_authority_list_anchor' test in mine, which may add some value.

https://github.com/ministryofjustice/laa-submit-crime-forms/pull/663/files#diff-caa3def9981603cfcc31966f2bbe59aba2777727934cb2f06a375510582584c1R59-R75